gym/f110_gym/envs/base_classes.py

```diff
@@ -59,7 +66,7 @@ class RaceCar(object):
     scan_angles = None
     side_distances = None
 
-    def __init__(self, params, seed, is_ego=False, time_step=0.01, num_beams=1080, fov=4.7):
+    def __init__(self, params, seed, is_ego=False, time_step=0.01, num_beams=1080, fov=4.7, integrator=Integrator.RK4):
```

Comment:
The default should probably stay Euler so it won't affect legacy code using the gym.